### ISSUE TITLE
could not download app onlyoffice

### ISSUE THREAD

**Issue by wqtb** [ONLYOFFICE/docker-onlyoffice-nextcloud#51]
When I use bash set_configruation.sh command
I get an error that could not download app onlyoffice
I don't know why

**Comment by igwyd** [comment]
Hello @wqtb
Do you followed the installation instructions https://github.com/ONLYOFFICE/docker-onlyoffice-nextcloud/blob/master/README.md ?
Please attach screenshot your error. What versions of docker and docker-compose you use?

**Comment by foxhoundv** [comment]
So I had the same issue, and it was basically because Nextcloud labeled OnlyOffice as untested for use with Nextcloud v21. So I had to login to my nextcloud instance and tell it to download the app. Then I had to enable it from the installed list. Then I could run the set_configuration.sh command.

**Comment by LinneyS** [comment]
@foxhoundv 
ONLYOFFICE/onlyoffice-nextcloud#419

**Comment by foxhoundv** [comment]
@LinneyS 
[https://github.com/ONLYOFFICE/onlyoffice-nextcloud/blob/master/CHANGELOG.md](url) When you look at the changelog for 6.3.0 it says it's compatible with Nextcloud 21

**Comment by LinneyS** [comment]
@foxhoundv
As you can see, this was released not long ago.
https://github.com/ONLYOFFICE/onlyoffice-nextcloud/releases/tag/v6.3.0

**Comment by foxhoundv** [comment]
@LinneyS Yes, I believe I was in that special moment of installing minutes to hours before they submitted the update. My original reply was more for a help for @wqtb so they knew it was possible. Thanks for your replies though.
